### ISSUE TITLE
Mastodon is not updated 

### ISSUE THREAD

**Issue by hanscees** [conda-forge/mastodon.py-feedstock#14]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

hi,

Mastodon on github is version 1.5 but on conda it is 1.2

something is broken
https://anaconda.org/conda-forge/mastodon.py/files

### Installed packages

```shell
not relevant
```

### Environment info

```shell
active environment : mastodon2022
    active env location : /home/hanscees/miniconda3/envs/mastodon2022
            shell level : 2
       user config file : /home/hanscees/.condarc
 populated config files : 
          conda version : 22.9.0
    conda-build version : not installed
         python version : 3.9.5.final.0
       virtual packages : __linux=5.15.0=0
                          __glibc=2.35=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/hanscees/miniconda3  (writable)
      conda av data dir : /home/hanscees/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/hanscees/miniconda3/pkgs
                          /home/hanscees/.conda/pkgs
       envs directories : /home/hanscees/miniconda3/envs
                          /home/hanscees/.conda/envs
               platform : linux-64
             user-agent : conda/22.9.0 requests/2.28.1 CPython/3.9.5 Linux/5.15.0-52-generic ubuntu/22.04.1 glibc/2.35
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

**Comment by jsta** [comment]
Version updating is blocked by the fact that `http_ece (http-ece)` is not on conda-forge.

**Comment by jsta** [comment]
http-ece is on conda-forge and the mastodon.py build is updated to the latest version. We can open a new issue if the build is somehow broken.
